### PR TITLE
core(category): add manualDescription

### PR DESCRIPTION
**Pull request by patrickhulce**
makes the change for manualDescription instead of a manual group

ref #5008 #4333

### CODE REVIEW

File: lighthouse-core/config/config.js [hunk 1]
```diff
@@ -62,7 +62,8 @@ function validateCategories(categories, audits, groups) {
         throw new Error(`could not find ${audit.id} audit for category ${categoryId}`);
       }
 
-      if (categoryId === 'accessibility' && !audit.group) {
+      const auditImpl = audits.find(a => a.implementation.meta.name === audit.id).implementation;
```
Comment (brendankenny) [inline lighthouse-core/config/config.js]: maybe get rid of `auditIds` above and use `auditImpl` for both this and the check of `audit.id` above?

Comment (patrickhulce) [reply lighthouse-core/config/config.js]: done

**Comment by patrickhulce** [comment]
@brendankenny feedback addressed!

**Comment by patrickhulce** [comment]
> should we add a check to validateCategories that any manual audit has a weight of 0?

ya mind if I do that after #5105 ? conflicting junk

**Comment by brendankenny** [comment]
> ya mind if I do that after #5105 ? conflicting junk

sure, but maybe add a TODO?

**Pull request merged**